### PR TITLE
configsync package

### PR DESCRIPTION
**Pull request by henderiw**
this is the package for configsync w/o rootsync which is in a separate repo

### CODE REVIEW

File: configsync/configsync.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: configmanagement.gke.io/v1
+kind: ConfigManagement
+metadata: # kpt-merge: /config-management
+  name: config-management
+  annotations:
+    config.kubernetes.io/depends-on: apiextensions.k8s.io/CustomResourceDefinition/configmanagements.configmanagement.gke.io
+    internal.kpt.dev/upstream-identifier: 'configmanagement.gke.io|ConfigManagement|default|config-management'
+spec:
+  enableMultiRepo: true
```
Comment (johnbelamaric) [inline configsync/configsync.yaml]: missing newline

File: configsync/package-context.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata: # kpt-merge: /kptfile.kpt.dev
+  name: kptfile.kpt.dev
+  annotations:
+    config.kubernetes.io/local-config: "true"
+    internal.kpt.dev/upstream-identifier: '|ConfigMap|default|kptfile.kpt.dev'
+data:
+  name: example
```
Comment (johnbelamaric) [inline configsync/package-context.yaml]: missing newline

**Review by johnbelamaric — APPROVED** [review]
/approve

non-blocking nits

File: configsync/configsync.yaml (hunk 1 above)
Comment (henderiw) [reply configsync/configsync.yaml]: fixed

File: configsync/package-context.yaml (hunk 2 above)
Comment (henderiw) [reply configsync/package-context.yaml]: fixed

**Pull request merged**